src/Consolonia.Core/Drawing/DrawingContextImpl.cs

```diff
@@ -731,14 +721,13 @@ private void DrawLineSymbolAndMoveHead(ref Point head, bool isVertical, ISymbol
 
         private void DrawStringInternal(IBrush foreground, string text, IGlyphTypeface typeface, Point origin = new())
         {
-            foreground = ConsoleBrush.FromBrush(foreground);
-            if (foreground is not ConsoleBrush { Mode: PixelBackgroundMode.Colored } consoleBrush)
+            if (foreground is not ISolidColorBrush solidColorBrush)
             {
                 ConsoloniaPlatform.RaiseNotSupported(4);
                 return;
             }
 
-            // if (!Transform.IsTranslateOnly()) ConsoloniaPlatform.RaiseNotSupported(15);
+            // if (!Transform.IsTranslateOnly()) ConsoloniaPlatform.RaiseNotSupported(15); //todo: what to do if a rotation?
```

Comment:
we could theoritcally handle 90, 180, 270, 360 rotations and flip x or y transforms.

Reply:
um...too much egg nog. I forgot FONT chars would have to transform as well. LOL